packages/amplication-data-service-generator/src/server/auth/guards/createDefaultGuard.ts

```diff
@@ -0,0 +1,61 @@
+import { builders } from "ast-types";
+import { print } from "recast";
+import { AUTH_PATH } from "../../constants";
+import { EnumAuthProviderType } from "../../../models";
+import { Module } from "../../../types";
+import {
+  addImports,
+  importNames,
+  interpolate,
+  removeTSClassDeclares,
+} from "../../../util/ast";
+import { readFile, relativeImportPath } from "../../../util/module";
+
+type AuthGuardMetaData = {
+  path: string;
+  fileName: string;
+  className: string;
+};
+
+export async function createDefaultGuard(
+  authProvider: EnumAuthProviderType
+): Promise<Module> {
+  const defaultAuthGuardPath = require.resolve(
+    "./defaultAuth.guard.template.ts"
+  );
+  const modulePath = `${AUTH_PATH}/defaultAuth.guard.ts`;
+
+  const templateGuardFile = await readFile(defaultAuthGuardPath);
+  const { path, className } = getMetaDataForAuthGuard(authProvider);
+  const baseGuardIdentifier = builders.identifier(className);
+  interpolate(templateGuardFile, {
+    GUARD: baseGuardIdentifier,
+  });
+  const baseGuardImport = importNames(
+    [baseGuardIdentifier],
+    relativeImportPath(modulePath, path)
+  );
+  addImports(templateGuardFile, [baseGuardImport]);
+  removeTSClassDeclares(templateGuardFile);
+  return { path: modulePath, code: print(templateGuardFile).code };
+}
+function getMetaDataForAuthGuard(
+  setAuthGuard: EnumAuthProviderType
+): AuthGuardMetaData {
+  const data: AuthGuardMetaData = { fileName: "", path: "", className: "" };
+  switch (setAuthGuard) {
+    case EnumAuthProviderType.Http:
+      data.className = "BasicAuthGuard";
+      data.fileName = "basicAuth";
+      data.path = `${AUTH_PATH}/${data.fileName}.guard.ts`;
+      break;
+    case EnumAuthProviderType.Jwt:
+      data.fileName = "jwt-auth";
```

Comment:
keep the same convention like basicAuth
```suggestion
      data.fileName = "jwtAuth";
```